### DISCUSSION TITLE
Rendering power/yield values in HTML `<title>` element?

### DISCUSSION THREAD

**Discussion by mjaschen** [tbnobody/OpenDTU#1101] [Ideas]
I usually have a browser tab open with OpenDTU. The `<title>` element currently shows only "OpenDTU".

What do you think about the idea to show here the current power or the value of the current day's yield? (maybe even configurable)

![screenshot-2023-06-30T143504-qdmju](https://github.com/tbnobody/OpenDTU/assets/328130/fbb04033-ae97-43fb-9df7-f57694b9402c)

**Comment by ituri** [comment]
+1!
